src/functions/template-tags/general.php

```diff
@@ -752,3 +752,105 @@ function tribe_context() {
 		return $context;
 	}
 }
+
+if ( ! function_exists( 'tribe_post_checksum' ) ) {
+	/**
+	 * Builds a post checksum based on a list of fields that are
+	 * set to change when relevant post details change.
+	 *
+	 * This function will only calculate the checksum: it will not perform any
+	 * kind of check or monitoring on the post itself.
+	 *
+	 * @since TBD
+	 *
+	 * @param int|WP_Post $post_id   The post ID or object
+	 * @param array       $use_frags An array detailing the post fields, columns of
+	 *                               the `posts` table, that should be used to build
+	 *                               the checksum; defaults to the post `ID` and `post_modified`
+	 *                               columns.
+	 *
+	 * @return null|string The checksum string or `null` if the post is not valid or no
+	 *                     fragments are passed.
+	 */
+	function tribe_post_checksum( $post_id, $use_frags = array( 'ID', 'post_modified' ) ) {
```

Comment:
```suggestion
	function tribe_post_checksum( $post_id, $use_frags = [ 'ID', 'post_modified' ] ) {
```

Also not sure if this default should be set in the function or within the function and check for `null === $use_frags`.